actionview/lib/action_view/digestor.rb

```diff
@@ -11,7 +11,7 @@ class << self
       #
       # * <tt>name</tt>         - Template name
       # * <tt>format</tt>       - Template format
-      # * <tt>finder</tt>       - An instance of <tt>ActionView::LookupContext</tt>
+      # * <tt>finder</tt>       - An instance of +ActionView::LookupContext+
```

Comment:
Do these need to change as well?
```suggestion
      # * +finder+       - An instance of +ActionView::LookupContext+
```

Reply:
I have another patch for symbols, wanted to take care of namespaces first